DigitalLearningSolutions.Web/ViewModels/LearningPortal/Current/MarkActionPlanResourceAsCompleteFormData.cs

```diff
@@ -20,6 +20,7 @@ protected MarkActionPlanResourceAsCompleteFormData(MarkActionPlanResourceAsCompl
         public int? Day { get; set; }
         public int? Month { get; set; }
         public int? Year { get; set; }
+        public bool ResourceSourcedFromFallbackData { get; set; }

```

Comment:
Think this should also be populated in the formData constructor. So the message is still displayed if the user gets shunted back to the page after validation failure. Will likely need some hidden input to include it in the form data as well.

Think we will need something similar on the Edit Complete By Date page.

Reply:
You're right, made those changes so the warning still shows if an error occurs